src/scenes/StageManager.ts

```diff
@@ -74,7 +74,16 @@ export default class StageManager{
         // transparent rendering mode is bad for overall performance, but necessary in order
         // to prevent flickering on some Android devices such as Galaxy Tab A and Kindle Fire
         const flickerProne = !!FLICKERERS.find((value) => value.test(navigator.userAgent));
-        this.pixi = new PIXI.Application({ width, height, antialias:true, transparent:flickerProne});
+
+        // Does this version of Safari break antialiasing?
+        let badSafari = navigator.userAgent.includes('Safari') && navigator.userAgent.includes('Version/15.4');
```

Comment:
we may want to modify this if there is a fix with 15.4.x since it looks like this will keep antialiasing off for all 15.4 versions. But good for now to push this out.